libs/util/git/src/constants.ts

```diff
@@ -16,3 +16,13 @@ The Amplication Team`;
 export const getDefaultREADMEFile = (repositoryName: string) => {
   return `# ${repositoryName}`;
 };
+
+export const getServicePullRequestMessage = (
+  serviceName: string | undefined,
+  serviceLink: string | undefined
+) => {
+  return `
+  If you're looking to bring in a new data model, tweak existing ones, handle APIs, or spice things up with Amplication plugins, just swing by the Amplication platform at [${serviceName}](${serviceLink}). Tell us what you need, and we'll handle all the coding magic for you.
+  
+  `;
+};
```

Comment:
@Meyazhagan as mention previously, in order to make this feature more flexible and keeping the util-git library generic and content agnostic, we would not expect any change to the util-git library in this PR. Please move this logic to the appropriate place (git-sync-manager).